galgebra/lt.py

```diff
@@ -611,17 +595,17 @@ def _latex(self, print_obj):
                 latex_str += ' & ' + coef_latex
             else:
                 latex_str += coef_latex
-            if cnt % self.lcnt == 0:
+            if cnt % lcnt == 0:
                 latex_str += '\\\\ '
                 cnt = 1
             else:
                 cnt += 1
-        if self.lcnt == len(expr_lst) or self.lcnt == 1:
+        if lcnt == len(expr_lst) or lcnt == 1:
             latex_str = latex_str[:-3]
         latex_str = latex_str + ' \\end{align*} \n'
         return latex_str
 
-    def Fmt(self, lcnt=1, title=None):
+    def Fmt(self, lcnt=1, title=None) -> printer._FmtResult:
```

Comment:
Any type returned by a public API is inevitably a part of the public API.

Reply:
Not if the constructor for that type is not part of the public API and it has no public members.

Reply:
Unless we did something really obnoxious like:
```
class FmtResult:
    def __new__(self, *args):
        raise TypeError("this type is private")

    @classmethod
    def _private_create(cls, title):
        return type.__new__(FmtResult, title)
```

which might not be an awful idea.

Reply:
The return type is usually a type within the type that is the only type that can create it (in C++).